src/batch_adder.rs

```diff
@@ -365,4 +365,20 @@ mod batch_add_tests {
             assert_eq!(buckets[i], tmp[i].add(points[i]));
         }
     }
+
+    #[test]
+    fn test_batch_add_indexed_single_bucket() {
+        let mut batch_adder = BatchAdder::new(1);
+        let mut rng = ark_std::test_rng();
+
+        let mut buckets: Vec<G1Affine> = vec![G1Affine::from(<G1Affine as AffineCurve>::Projective::rand(&mut rng))];
+        let points: Vec<G1Affine> = (0..3)
```

Comment:
nit, I assume 3 is a random number picked? Probably better to create a single point vector as it requires less guessing.

Reply:
sure, I've updated it